packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -24,7 +24,7 @@ export const StorageImage = ({
   const options = React.useMemo(
     () => ({
       accessLevel,
-      identityId,
+      targetIdentityId: identityId,
```

Comment:
A bit more work, but think a holistic approach to addressing the issue would be: 
- add `targetIdentityId` as a prop on `StorageImage`
- add a deprecation warning for usage of `identityId` (JSDoc and with `useDeprecationWarning`)
- update the params of `useStorageURL` to accept `targetIdentityId` in place of `identityId`